gcp_variant_transforms/transforms/partition_variants_test.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2017 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for partition_variants module."""
+
+from __future__ import absolute_import
+
+import unittest
+
+from apache_beam import Partition
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+from apache_beam.transforms import Create
+
+from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms.transforms import partition_variants
+
+
+class PartitionVariantsTest(unittest.TestCase):
+  """Test cases for the ``PartitionVariants`` transform."""
+
+  def _get_valid_variants(self):
+    # Valid varints reference_name.strip().lower() will successfully matched to
+    #   re.compile(r'^(c|ch|chr)?([0-9][0-9]?)$')
+    variants = []
+    # Partition 0
+    variants.append(vcfio.Variant(reference_name='chr1'))
+    variants.append(vcfio.Variant(reference_name='CHR1'))
+    # Partition 1
+    variants.append(vcfio.Variant(reference_name='ch2'))
+    variants.append(vcfio.Variant(reference_name='CH2'))
+    # Partition 2
+    variants.append(vcfio.Variant(reference_name='c3'))
+    variants.append(vcfio.Variant(reference_name='C3'))
+    # Partition 5
+    variants.append(vcfio.Variant(reference_name='6'))
+    variants.append(vcfio.Variant(reference_name='06'))
+    variants.append(vcfio.Variant(reference_name='chr06'))
+    # Partition 9
+    variants.append(vcfio.Variant(reference_name='chr10'))
+    variants.append(vcfio.Variant(reference_name='  chr10   '))
+    # Partition 21
+    variants.append(vcfio.Variant(reference_name='chr22'))
+    variants.append(vcfio.Variant(reference_name='chr22    '))
+    return variants, variants[0:2], variants[2:4], variants[4:6], variants[6:9], variants[9:11], variants[11:13]
+
+  def _get_invalid_variants(self):
```

Comment:
consider rewording 'valid' and 'invalid'  (perhaps 'nonstandard' and 'standard'?)